rero_mef/authorities/api.py

```diff
@@ -87,13 +87,121 @@ class MefRecord(AuthRecord):
     fetcher = mef_id_fetcher
     provider = MefProvider
 
+    @classmethod
+    def build_ref_string(cls, agency_pid, agency):
+        """Build ref string."""
+        from flask import current_app
+        with current_app.app_context():
+            url = current_app.config.get('JSONSCHEMAS_HOST')
+            ref_string = 'http://' + url + '/api/' + agency + '/' + agency_pid
+            return ref_string
+
+    @classmethod
+    def get_mef_by_agency_pid(cls, agency_pid, agency, with_deleted=False):
+        """Get mef record by agency pid value."""
+        key = agency + '.identifier_for_person'
+        search = MefSearch()
+        result = (
+            search.filter('term', **{key: agency_pid})
+            .source()
+            .execute()
+            .to_dict()
```

Comment:
do not use to_dict, but `scan`. See https://github.com/rero/rero-ils/blob/54b781457d68216f6fb296a3a2e16e2a0767827b/rero_ils/modules/members/api.py#L54 for examples.